package-res/resources/web/pentaho/data/filter/Not.js

```diff
@@ -56,6 +56,11 @@ define([
    *  var filter = new Filter.Not(sales12k);
    *  var data = filter.apply(data); //data.getValue(0, 0) === ["B", "D", "E", "F", "G"]
```

Comment:
No. data.getValue(0, 0) returns a single value. You meant:
/*
data.getValue(0, 0) === "B"
...
data.getValue(4, 0) === "G"
*/